x/avs/keeper/avs.go

```diff
@@ -37,25 +54,74 @@ func (k *Keeper) GetAVSMinimumSelfDelegation(ctx sdk.Context, avsAddr string) (s
 		return sdkmath.LegacyNewDec(0), errorsmod.Wrap(err, fmt.Sprintf("GetAVSMinimumSelfDelegation: key is %s", avsAddr))
 	}
 
-	return sdkmath.LegacyNewDec(avsInfo.Info.MinSelfDelegation.Int64()), nil
+	return sdkmath.LegacyNewDec(int64(avsInfo.Info.MinSelfDelegation)), nil
 }
 
 // GetEpochEndAVSs returns the AVS list where the current block marks the end of their epoch.
-func (k *Keeper) GetEpochEndAVSs(ctx sdk.Context) ([]string, error) {
-	var avsList []types.AVSInfo
-	k.IterateAVSInfo(ctx, func(_ int64, epochEndAVSInfo types.AVSInfo) (stop bool) {
-		avsList = append(avsList, epochEndAVSInfo)
+func (k *Keeper) GetEpochEndAVSs(ctx sdk.Context, epochIdentifier string, epochNumber int64) ([]string, error) {
```

Comment:
If this is never returning an error, we should remove the type from the returned items.

Reply:
fixed